Code/GraphMol/Chirality.cpp

```diff
@@ -1068,6 +1075,10 @@ void assignStereochemistry(ROMol &mol, bool cleanIt, bool force,
   }
 
   if (cleanIt) {
+    // if the ranks are needed again, this will force them to be
+    // re-calculated based on the stereo calculated above.
+    atomRanks.clear();
```

Comment:
Do you have a test-case that demonstrates the necessity of this call? I'm having a hard time seeing it and commenting it out doesn't break the tests you currently have checked in.